popup.js

```diff
@@ -166,6 +234,22 @@ const setAllocationsAndConfirmations = () => {
   }
 };
 
+const getCombinationLabel = (candidates, ordinal) => {
+  const isControl = ordinal === candidates[0].ordinal;
```

Comment:
This assumes the candidate that is firs tin the list is control. I'm not sure we can guarantee that. We do know it is the combo with the lowest ordinal

Reply:
I've added sort.